index.html

```diff
@@ -0,0 +1,147 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Gentium+Plus:wght@700&family=Poppins&family=Roboto:wght@700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./styles/styles.css">
+</head>
+
+<body>
+    <nav class="sub-nav bg-secondary">
+        <div class="container">
+            <ul class="sub-nav-list float-right">
+                <li class="sub-nav-item"><i class="fa-brands fa-facebook-f"></i></li>
+                <li class="sub-nav-item"><i class="fa-brands fa-twitter"></i></li>
+                <li class="sub-nav-item">English</li>
+                <li class="sub-nav-item">My Page</li>
+                <li class="sub-nav-item">Logout</li>
+            </ul>
+        </div>
+    </nav>
+    <header class="navbar fixed-top-nav" id="top-header">
+        <div class="container">
+            <i class="fa-solid fa-bars" id="menu-icon"></i>
+            <div class="desktop-nav">
+                <img class="logo-nav" src="./images/aspire-logo.png" width="110px" alt="Aspire logo">
+                <ul class="nav-list float-right">
+                    <li><a class="menu-list-item" href="about.html">About</a></li><br>
+                    <li><a class="menu-list-item" href="#about-myself">Program</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">Join</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">Sponsor</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">News</a></li><br>
+                    <li><a href="#" class="mt-3  p-2" id="btn-nav">Company</a></li><br>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <nav class="mobile-nav">
+        <span class="close-button">x</span>
+        <ul class="menu-list-mobile">
+            <li><a class="menu-list-item" href="about.html">About</a></li><br>
+            <li><a class="menu-list-item" href="#main-program">Program</a></li><br>
+            <li><a class="menu-list-item" href="#join">Join</a></li><br>
+            <li><a class="menu-list-item" href="#sponsor">Sponsor</a></li><br>
+            <li><a class="menu-list-item" href="#news">News</a></li><br>
+            <li><a href="#" class="mt-3  p-2" id="btn-nav">BP Launch</a></li><br>
+        </ul>
+    </nav>
+
+    <main class="main-section">
+        <section class="top-bg">
+            <section class="hero container">
+                <br>
+                <h4 class="greetings">"Explore Your Dream Career with Aspire"</h4>
+                <h2 class="hero-title">Get connected <br /> to the experts Globally.</h2>
```

Comment:
Please let's consider using an `h1` tag in each page for the main title i.e `Get connected
to the experts ...` 

The H1 tag is important for SEO, accessibility and usability, so ideally, you should have one on each page of your site.